test/pageobjects/DocumentUpload.js

```diff
@@ -32,11 +33,23 @@ class DocumentUpload extends BasePage {
     // Input here cannot use the uploadInput() function above
     const input = this.$('.onfido-sdk-ui-CustomFileInput-input')
     const pathToTestFiles = '../resources/'
-    // This will detect local file, ref: https://www.browserstack.com/automate/node#enhancements-uploads-downloads
-    this.driver.setFileDetector(new remote.FileDetector())
-    const sendKeysToElement = input.sendKeys(
-      path.join(__dirname, pathToTestFiles + filename)
-    )
+    //the below line...if safari AND Local, ignore it...otherwise file uploads will fail.
+    if (browserName === 'safari' && isRemoteBrowser === false) {
```

Comment:
Can `isRemoteBrowser` be undefined or null?

Reply:
Nope, it cant be, it has to be explicitly set in the config...(take a look at config.json)

"remote": false or "remote": true